src/stream_base.cc

```diff
@@ -102,8 +102,7 @@ int StreamBase::Writev(const FunctionCallbackInfo<Value>& args) {
 
   size_t count = chunks->Length() >> 1;
 
-  uv_buf_t bufs_[16];
-  uv_buf_t* bufs = bufs_;
+  MaybeStackBuffer<uv_buf_t> bufs(count);
```

Comment:
sure, but still pass in `count` when creating it as I'm doing?

Or should I reinstate l135-137 to create the bigger buffer if required?